<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlDesigner::AnnotationCommentTab</class>
 <widget class="QWidget" name="QmlDesigner::AnnotationCommentTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="titileLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="authorEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>authorEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
